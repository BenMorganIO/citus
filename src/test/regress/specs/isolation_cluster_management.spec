session "s1"
step "s1a"
{
    SELECT 1 FROM master_add_node('localhost', 57637);
    SELECT 1 FROM master_add_node('localhost', 57638);
    SELECT nodename, nodeport, isactive FROM pg_dist_node ORDER BY nodeid;
}

permutation "s1a"
