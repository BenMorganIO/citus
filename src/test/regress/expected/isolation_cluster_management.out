Parsed test spec with 1 sessions

starting permutation: s1a
step s1a: 
    SELECT 1 FROM master_add_node('localhost', 57637);
    SELECT 1 FROM master_add_node('localhost', 57638);
    SELECT nodename, nodeport, isactive FROM pg_dist_node ORDER BY nodeid;

?column?       

1              
?column?       

1              
nodename       nodeport       isactive       

localhost      57637          t              
localhost      57638          t              
